database/migrations/2024_06_11_220722_update_field_length.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('activity_log_subjects', function (Blueprint $table) {
+            $table->string('subject_type', 255)->change();
```

Comment:
Omit 255 as the parameter for each of them, it's the default!

Reply:
I tried without specifying it won't overwrite 191.